widget/embedded/src/components/CustomTokenModal/CustomTokenModal.types.ts

```diff
@@ -0,0 +1,9 @@
+import type { BlockchainMeta, Token } from 'rango-sdk';
+
+export type PropTypes = {
+  open: boolean;
+  onClose: () => void;
+  handleSubmitClick: () => void;
+  token?: Token;
```

Comment:
Why this is optional?

Reply:
I fixed it.
I use this condition `selectedBlockchainForCustomToken && token` to open modals.